### ISSUE TITLE
(Exclusively for auto generated captions on YT) Max mute time feature

### ISSUE THREAD

**Issue by CreeperPookie** [richardfrost/AdvancedProfanityFilter#199]
With the YT auto-generated caption support, it made your extension improve tenfold. However, one thing I noticed was that sometimes the captions wouldn't detect the next word that was said, and therefore would stay muted for a long period of time, and could be quite annoying at times.

#### :twisted_rightwards_arrows: Steps To Reproduce
1. Find a video that does this (Example: https://youtu.be/tmIZOwYRS3c?t=364 ; note: you must have the word "God" filtered to something)
2. Watch how it doesn't unmute untill 20+ seconds later

#### :heavy_check_mark: Expected behavior
Instead of this, what it should do is have a maximum mute time for this. However, for real captions,  it doesn't really apply; only for auto-generated

#### :computer: Details
 - **Device:** Dell Inspiron 17-7779
 - **OS:** Windows 10
 - **Browser:** Microsoft Edge (Chromium), using chrome APF extension
 - **Browser Version:** 80.0.361.69
 - **Affected site(s):** youtube.com

**Comment by richardfrost** [comment]
This has now been added to version [2.7.0](https://github.com/richardfrost/AdvancedProfanityFilter/releases/tag/2.7.0) Please give it a try and report back how it works. Thanks!

_2.7.0 has been submitted to all the browser stores, but each must be approved before it is available. Firefox has already been approved._

**Comment by richardfrost** [comment]
This is now released and should be added. Feel free to re-open or create a new issue if it doesn't work for you.
